pkg/vfs/accesslog.go

```diff
@@ -31,6 +33,10 @@ var (
 		Help:    "Operations latency distributions.",
 		Buckets: prometheus.ExponentialBuckets(0.0001, 1.5, 30),
 	}, []string{"method"})
+	opsIOErrors = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "fuse_ops_io_errors",
+		Help: "Number of IO errors.",
+	}, []string{"errno"})
```

Comment:
We should add the `errno` into the name of metric in `.stats`, see internal.go

Reply:
Do you mean change `juicefs_fuse_ops_io_errors` to `juicefs_fuse_ops_io_errors_errno` in .stats file?

Reply:
yes

Reply:
Added